src/LockFreeIndexSetBuilders.cpp

```diff
@@ -141,8 +141,12 @@ void buildLockFreeBlockIndexset(RAJA::IndexSet& iset,
       }
     }
   } else { /* 3d mesh */
+
```

Comment:
We're going to get rid of these index set builder routines since they are not general and were developed for exploring ideas in LULESH.  We will move them to the proxy app repo as part of cleaning that up.